project/src/main/webapp/event-details.html

```diff
@@ -3,10 +3,15 @@
   <head>
     <meta charset="UTF-8" />
     <title>STEP 188 Project</title>
+    <link rel="stylesheet" href="event-details.css" />
+    <script src="event-details.js"></script>
     <script src="script.js"></script>
     <link rel="stylesheet" href="styles.css" />
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js"></script>
+    <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
There are couple of repeated scripts here, I think: script.js, event-details.js, and jquery
(might also be a thing in other HTML files that I didn't catch)

Reply:
Also JQuery should go before all our local scripts (no need to fix here bc I have a PR with those fixes - but if you see merge conflicts about this, choose the one with JQuery before our scripts)

Reply:
Ack, sg